bin/sensei.js

```diff
@@ -0,0 +1,124 @@
+#! /usr/bin/env node
+const spawn = require("child_process").spawn;
+const fork = require("child_process").fork;
+const path = require("path");
+const WebpackDevServer = require("webpack-dev-server");
+const config = require("../webpack.config");
+const webpack = require("webpack");
+
+function cli(args) {
+  let trainingMaterialFolder = args[3];
+  switch (args[2]) {
+    case "serve":
+    case null:
```

Comment:
This behavior is a little bit confusing : 

```shell
sensei
```
will serve the current dir

```shell
sensei ../another-training
```
will display help

I think it is better to display help when no params is specified, WDYT?